src/main/java/org/deepsymmetry/Wayang.java

```diff
@@ -73,11 +73,22 @@
     private static BufferedImage displayImage = null;
 
     /**
+     * Transfer object for asynchronous calls.
+     */
+    private static Transfer transfer = null;
```

Comment:
I am moving this to a pair of local variables in `sendFrameAsync()` because, since it is asynchronous, there may be more than one transfer going on at once. In fact, even when sending just one frame, the header transfer may still be taking place when the pixel data transfer is started.